src/EditorFeatures/Core/LanguageServer/AbstractInProcLanguageClient.cs

```diff
@@ -267,6 +268,13 @@ public Task OnServerInitializedAsync()
     /// </summary>
     public Task AttachForCustomMessageAsync(JsonRpc rpc) => Task.CompletedTask;
 
+    private static PropertyCollection CreateStjPropertyCollection()
```

Comment:
>CreateStjPropertyCollection [](http://example.com/codeflow?start=38&length=27)

the vslanguageserverclient code indicates this is a temporary measure for 17.10. Is there going to be a better way to do this soon?

[https://devdiv.visualstudio.com/DevDiv/_git/VSLanguageServerClient?path=/src/product/RemoteLanguage/Impl/Infrastructure/MefClientProvider/RemoteLanguageClientInstance.cs&version=GBdevelop&line=481&lineEnd=482&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents](https://devdiv.visualstudio.com/DevDiv/_git/VSLanguageServerClient?path=/src/product/RemoteLanguage/Impl/Infrastructure/MefClientProvider/RemoteLanguageClientInstance.cs&version=GBdevelop&line=481&lineEnd=482&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents)